src/main/resources/data/alaskanativecraft/recipes/purple_dye_from_salmonberries.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "minecraft:crafting_shapeless",
+  "group": "purple_dye",
+  "ingredients": [
+    {
+      "item": "alaskanativecraft:salmonberries"
+    }
+  ],
+  "result": {
+    "item": "minecraft:purple_dye"
+  }
+}
```

Comment:
Same comment as the cloudberry variant of this recipe. Also perhaps a tag would serve here as they are very similar: cloudberries are usually called salmonberries too